### PR TITLE
TCVP-1058 Add MassTransit to staff-api

### PR DESCRIPTION
**Pull request by KDNeufeld**
# Description

This PR includes the following proposed change(s):

[TCVP-1058](https://justice.gov.bc.ca/jira/browse/TCVP-1058)

- Added MassTransit (RabbitMQ) to staff-api project
- Converted DisputeApproved message contract to a class from an interface
- Added DisputeCancelled message contract
- Added MassTransit events to reject/cancel/submit endpoints

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

dotnet test

## Does the change impact or break the Docker build?

- [x] Yes
- [ ] No

If Yes: Has Docker been updated accordingly?

- [x] Yes
- [ ] No

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/backend/TrafficCourts/Staff.Service/Services/DisputeService.cs [hunk 1]
```diff
@@ -51,20 +56,36 @@ public async Task<Dispute> UpdateDisputeAsync(Guid disputeId, Dispute dispute, S
 
     public async Task CancelDisputeAsync(Guid disputeId, CancellationToken cancellationToken)
     {
+        _logger.LogDebug("Dispute cancelled.");
```
Comment (pbolduc) [inline src/backend/TrafficCourts/Staff.Service/Services/DisputeService.cs:59]: log messages dont need periods. Consider them a phrase, not a complete sentence.

**Review by pbolduc — APPROVED** [review]
I had some comments, but nothing blocking the merge of this.

File: src/backend/TrafficCourts/Staff.Service/Services/DisputeService.cs (hunk 1 above)
Comment (KDNeufeld) [reply src/backend/TrafficCourts/Staff.Service/Services/DisputeService.cs]: Done

File: src/backend/TrafficCourts/Staff.Service/Mappers/Mapper.cs [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+namespace TrafficCourts.Staff.Service.Mappers;
+
+public class Mapper
+{
+    public static DisputeApproved ToDisputeApproved(Dispute dispute)
+    {
+        DisputeApproved target = new();
+        target.CitizenName = dispute.GivenNames + " " + dispute.DisputantSurname;
+        target.TicketIssuanceDate = dispute.ViolationDate.UtcDateTime;
```
[the remark this answers is not in the document]
Comment (KDNeufeld) [reply src/backend/TrafficCourts/Staff.Service/Mappers/Mapper.cs]: Changed to dispute.ViolationDate.DateTime.  You're right, ViolationDate is non-timezone specific.

**Review by lfdgov — APPROVED** [review]

**Pull request merged**